### PR TITLE
[SW-1132] Remove usages of PROTOCOL_AETERNITY as default value in composables

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/composables/currencies.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const initPollingWatcher = createPollingBasedOnMountedComponents(POLLING_INTERVA
 export function useCurrencies({
   withoutPolling = false,
   store,
-  selectedProtocol = PROTOCOL_AETERNITY, // TODO - remove default value & make the protocol required
+  selectedProtocol = PROTOCOL_AETERNITY,
```
Comment (pstachel) [inline src/composables/currencies.ts:56]: I didn't make any changes in this PR to this file as it is tackled in #2327

Comment (peronczyk) [inline src/composables/currencies.ts:56]: I suggest to rename this into `protocol`. "selected" adds no context here.

Comment (pstachel) [reply src/composables/currencies.ts:56]: this will be removed in #2327

File: src/popup/components/AccountSelectOptionsItem.vue [hunk 2]
```diff
@@ -70,9 +72,16 @@ export default defineComponent({
         : 0,
     );
 
+    const tokenSymbol = computed(
+      () => (props.account.protocol)
```
Comment (peronczyk) [inline src/popup/components/AccountSelectOptionsItem.vue:76]: The `protocol` property is required in the account and it is always set so there is no need to check it.

File: src/popup/components/SwapRates.vue [hunk 3]
```diff
@@ -57,6 +59,9 @@ export default {
   props: {
     transaction: { type: Object, required: true },
   },
+  data: () => ({
+    PROTOCOL_AETERNITY,
+  }),
```
Comment (peronczyk) [inline src/popup/components/SwapRates.vue:64]: Please use setup here.

File: src/popup/components/TransactionDetailsBase.vue [hunk 4]
```diff
@@ -196,7 +196,8 @@
           <template #value>
             <TokenAmount
               :amount="+gasPrice"
-              symbol="AE"
+              :protocol="PROTOCOL_AETERNITY"
+              :symbol="AE_SYMBOL"
```
Comment (peronczyk) [inline src/popup/components/TransactionDetailsBase.vue:200]: This is a "Base" component so in my opinion passing the AE related constants here looks off. Maybe the parent should pass this down as a prop?

File: src/popup/pages/Names/AuctionBid.vue [hunk 5]
```diff
@@ -79,6 +82,7 @@ import {
 } from '@/protocols/aeternity/config';
 import { aeToAettos } from '@/protocols/aeternity/helpers';
 
+import { PROTOCOL_AETERNITY } from '@/constants';
```
Comment (peronczyk) [inline src/popup/pages/Names/AuctionBid.vue:85]: Please move this below the types import.

File: src/popup/pages/Names/AuctionHistory.vue
Comment (peronczyk) [inline src/popup/pages/Names/AuctionHistory.vue:1]: Maybe also convert the script block into TS?

File: src/protocols/aeternity/views/TransactionDetails.vue [hunk 6]
```diff
@@ -92,6 +93,7 @@ import DetailsItem from '@/popup/components/DetailsItem.vue';
 import LinkButton from '@/popup/components/LinkButton.vue';
 import Truncate from '@/popup/components/Truncate.vue';
 import CopyText from '@/popup/components/CopyText.vue';
+import { PROTOCOL_AETERNITY } from '@/constants';
```
Comment (peronczyk) [inline src/protocols/aeternity/views/TransactionDetails.vue:96]: Please move this line under the types import.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/TransactionDetailsBase.vue (hunk 4 above)
Comment (pstachel) [reply src/popup/components/TransactionDetailsBase.vue:200]: This component needs refactoring as it has some protocol specific things - I didn't want to tackle this in this PR

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/AccountCardMultisig.vue [hunk 7]
```diff
@@ -11,7 +11,10 @@
     </template>
 
     <template #middle>
-      <BalanceInfo :balance="+account.balance" />
+      <BalanceInfo
+        :balance="+account.balance"
+        :protocol="PROTOCOL_AETERNITY"
```
Comment (konradodo) [inline src/popup/components/AccountCardMultisig.vue:16]:
```suggestion
        :protocol="account.protocol"
```

Comment (pstachel) [reply src/popup/components/AccountCardMultisig.vue:16]: There should be PROTOCOL_AETERNITY selected as multisig views are using AE protocol

**Pull request merged**